### PR TITLE
Connect 4 Project Expansion

### PR DESCRIPTION
**Pull request by rtillies**
# Backend Curriculum PR Template

### Description

*Provide a brief summary of the change(s) included in this pull request*
Expanded Connect Four project

### Why are we making this update?
To expand Connect Four project as a viable Mod 1 pair project for use in 2201 inning

*Share the reason(s) for this pull request*
  
### Type of update

- [ ] Minor update/fix -- no review requested
- [X] Moderate update -- review from Mod Team requested
- [ ] Major update -- review from Backend Team requested

### How will we measure the success of this change? 

*Share any hypothesized outcomes we might observe*
Student pairs will be able to complete the Connect Four project with more precision 

### What questions do you have/what do you want feedback on? (optional)

*List any specific questions and feedback requests*
projects/connect_four.md is the current status of Connect Four (single page, broken Rubric link)
projects/connect_four directory is more fleshed out, similar to Battleship (projects/battleship): multiple pages, Overview and Requirements sections, expanded Iterations
I was waffling between using Mastermind or Connect Four for this inning's pair project. They are both in similar states of thoroughness. The Pair Project kickoff is tomorrow (Monday, February 7) and the project is due on Wednesday, February 16.

### CODE REVIEW

File: module1/projects/connect_four/iteration_2.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+In this iteration, you are required to use TDD to create your classes. Use the interaction pattern to determine what a method should do and write one or more tests to verify that expected behavior. Then you can implement the method. You should always write code with the purpose of making a test pass.
+
+## Placing Pieces
+
+Update your program to request user input and allow them to place an individual piece.
+
+Your program should ask for the user to enter a letter A - G, update the board so that their piece has been placed in that column, and then display the board again.
+
+Your program should ask the user to enter a letter A-G.
+Update the board to display the player's piece in the lowest available position of the selected column with an 'X'.
```
Comment (cjsim89) [inline module1/projects/connect_four/iteration_2.md:18]: Lines 17-20 repeat the same information... I think lines 19-20 are probably the most descriptive and should be kept.

**Review by cjsim89 — APPROVED** [review]
Besides the one grammatical flow suggestion I made in iteration 2, this looks good to me

File: module1/projects/connect_four/iteration_2.md (hunk 1 above)
Comment (rtillies) [reply module1/projects/connect_four/iteration_2.md:18]: Fixed, thanks for catching that!

**Review by cjsim89 — APPROVED** [review]

**Review by megstang — APPROVED** [review]
This all looks good to me. Excited for this project change! Thanks for doing this, Richard

**Pull request merged**